### DISCUSSION TITLE
FlowmapBlue - Colorcoding of different flows becomes only visible when zooming into map

### DISCUSSION THREAD

**Discussion by AnkaNova** [FlowmapBlue/FlowmapBlue#94] [Q&A] [answered]
I added a colorcoding to the flows that assigns a different color depending on the various entry points of the flows.
But the colorcoding only works when I start to zoom into the map (see several screenshots). Is there a way to fix this?

![Screenshot1_Zoomout](https://github.com/FlowmapBlue/FlowmapBlue/assets/140158937/f713213f-23f6-4b37-98e2-915f0cad8538)
![Screenshot2_Zoomin1](https://github.com/FlowmapBlue/FlowmapBlue/assets/140158937/66fcf3ac-c8b3-444f-a9aa-ed4aca1de654)
![Screenshot3_Zoomin2](https://github.com/FlowmapBlue/FlowmapBlue/assets/140158937/007f5aa5-1845-4a99-a1d4-1328f8991ea0)

**Comment by ilyabo** [comment, marked as answer]
Hi Anka! Sorry for the delay. Currently the flow coloring only works for unclustered flows because the clustered ones might contain flows of multiple colors. So I think it makes sense to disable clustering (and maybe also "Location totals") in the UI when using colored flows.

<img width=150 src=https://github.com/FlowmapBlue/FlowmapBlue/assets/351828/6a9f26c5-401b-4912-ae98-2603ea52f5fe>
